### PR TITLE
introduce translation comments, update trans. + german

### PR DESCRIPTION
**Pull request by theGreatWhiteShark**
@mauser I've introduced some comments describing the meaning of the translatable strings in the context of the session management (and a few others).

### CODE REVIEW

File: data/i18n/hydrogen_fr.ts [hunk 1]
```diff
@@ -3183,6 +3182,7 @@ Le chemin vers le script et le nom du script doivent être sans espaces.</transl
     </message>
     <message>
         <source>Use &amp;LASH</source>
+        <extracomment>LASH (acronym for LASH Linux Audio Handler) is a protocol allowing for session management in Linux (recalling songs, preferences, and JACK port connections).</extracomment>
```
Comment (trebmuh) [inline data/i18n/hydrogen_fr.ts:3185]: LASH = Linux Audio Session Handler (if I'm not wrong)

**Review by trebmuh — CHANGES_REQUESTED** [review]
Nice move !

File: data/i18n/hydrogen_fr.ts (hunk 1 above)
Comment (theGreatWhiteShark) [reply data/i18n/hydrogen_fr.ts:3185]: Your right, I mixed that up. But according to the https://wiki.linuxaudio.org/apps/categories/lash it's a recursive one `LASH Audio Session Handler`

Comment (trebmuh) [reply data/i18n/hydrogen_fr.ts:3185]: That is right : [http://git.savannah.nongnu.org/cgit/lash.git/tree/README].

**Pull request merged**